api/client/gro_client.py

```diff
@@ -302,8 +302,24 @@ def get_geo_centre(self, region_id):
         """
         return lib.get_geo_centre(self.access_token, self.api_host, region_id)
 
+
+    def get_geojsons(self, region_id, descendant_level=None):
+        """Given a region ID, return shape information in geojson, for the
+        region and all its descendants at the given level (if specified).
```

Comment:
add valid list of descendant_levels. admin levels 2~5. getting api response error:
`{"statusCode":400,"error":"Bad Request","message":"child \"reqRegionLevelId\" fails because [\"reqRegionLevelId\" must be less than or equal to 5]","validation":{"source":"query","keys":["reqRegionLevelId"]}}`

Reply:
done (although we should generalize it on the server side to include 8)